src/docs/product/relay/options.mdx

```diff
@@ -6,7 +6,7 @@ redirect_from:
   - /meta/relay/options/
 ---
 
-Configuration for Relay is recorded in the file `.relay/config.yml`. To change this location, pass the `--config` option to any Relay command:
+Configuration for Relay is recorded either in the file `.relay/config.yml`, or when noted, will be overridden through an environment variable. To change the config location, pass the `--config` option to any Relay command:
```

Comment:
@jan-auer  suggested that the docs should list the options that can be chosen from environment variables. Since this page doesn't show all the relay config options (by design i assume), I've only noted on the current shown options where they can be also configured by env-vars. 

I'm looking for feedback on whether we should have a separate page for all the environmental variables regardless, or keep it like this.